tests/test_gpu_examples.py

```diff
@@ -1383,6 +1384,87 @@ def test_model_loaded_in_float16_working(self):
             trainer.train()
 
 
+@require_torch_gpu
+@require_aqlm
+class PeftAqlmGPUTests(unittest.TestCase):
+    r"""
+    AQLM + peft tests
+    """
+
+    def setUp(self):
+        self.causal_lm_model_id = "BlackSamorez/TinyLlama-1_1B-Chat-v1_0-AQLM-2Bit-1x16-hf"
```

Comment:
This model is stored in a pickle file, for tests we should really move to safetensors. Would it be possible for you to convert it or switch to a safetensors model for testing? Also, we should move models used for testing over to https://huggingface.co/peft-internal-testing, which I can do once we have a safetensors model.